### ISSUE TITLE
[Question] the FIXME in llava_arch.py

### ISSUE THREAD

**Issue by henrycjh** [haotian-liu/LLaVA#558]
### Question

Hi， I am curious about the reason you do this fix. I think cur_image_features[0:0] is just an empty tensor.
`
  #FIXME: this is a hacky fix, for deepspeed zero3 to work
`
`
  cur_input_embeds = torch.cat([cur_input_embeds_1, cur_image_features[0:0], cur_input_embeds_2], dim=0)
`

**Comment by hangzeli05** [comment]
我也有这个疑问，请问解决了吗
